lib/msf/core/post/file.rb

```diff
@@ -578,6 +579,18 @@ def copy_file(src_file, dst_file)
 
 protected
 
+  def _read_file_powershell(file_name)
+    puts file_name
```

Comment:
```suggestion
```

Reply:
sorry forgot to remove debug statements :sweat_smile: